### PR TITLE
fix tiup example yaml link

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Signed-off-by: zhangjinpeng1987 <zhangjinpeng@pingcap.com>

### What is changed, added or deleted? (Required)

Fix tiup example yaml link.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap-incubator/tiup-cluster/pull/343

### CODE REVIEW

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ category: how-to
 
 ## 第 4 步：配置初始化参数文件 `topology.yaml`
 
-集群初始化配置文件需要手动编写，完整的全配置参数模版可以参考 [Github TiUP 项目配置参数模版](https://github.com/pingcap-incubator/tiops/blob/master/topology.example.yaml)。
+集群初始化配置文件需要手动编写，完整的全配置参数模版可以参考 [Github TiUP 项目配置参数模版](https://github.com/pingcap-incubator/tiops/blob/master/examples/topology.example.yaml)。
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:296]:
```suggestion
集群初始化配置文件需要手动编写，完整的全配置参数模版可以参考 [Github TiUP 项目配置参数模版](https://github.com/pingcap-incubator/tiup-cluster/blob/master/examples/topology.example.yaml)。
```
我看这个链接重定向到 https://github.com/pingcap-incubator/tiup-cluster/blob/master/examples/topology.example.yaml 了，不如直接写这个链接？

Comment (zhangjinpeng1987) [reply how-to/deploy/orchestrated/tiup.md:296]: good catch

Comment (zhangjinpeng1987) [reply how-to/deploy/orchestrated/tiup.md]: done

**Comment by yikeke** [comment]
LGTM

**Pull request merged**